src/EventBuilder/Cecil/StaticEventTemplateInformation.cs

```diff
@@ -0,0 +1,158 @@
+﻿using EventBuilder.Entities;
+using Mono.Cecil;
+using System.Collections.Generic;
+using System.Linq;
+
+namespace EventBuilder.Cecil
+{
+    public static class StaticEventTemplateInformation
+    {
+        private static readonly Dictionary<string, string> SubstitutionList = new Dictionary<string, string>
+        {
+            {"Windows.UI.Xaml.Data.PropertyChangedEventArgs", "global::System.ComponentModel.PropertyChangedEventArgs"},
+            {
+                "Windows.UI.Xaml.Data.PropertyChangedEventHandler",
+                "global::System.ComponentModel.PropertyChangedEventHandler"
+            },
+            {"Windows.Foundation.EventHandler", "EventHandler"},
+            {"Windows.Foundation.EventHandler`1", "EventHandler"},
+            {"Windows.Foundation.EventHandler`2", "EventHandler"}
+        };
+
+        private static string RenameBogusWinRTTypes(string typeName)
```

Comment:
Given this is only being used for Xamarin.Essentials, do we need this slightly odd bodge?

Reply:
I am going to kick it all out for now.